### PR TITLE
stolostron/backlog#22678 ki CoreOS node

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -930,3 +930,11 @@ View the following restore limitations:
 * The new hub cluster has a different configuration from the active hub cluster when a user reapplies the backup on the new hub cluster. Since there is an existing policy on the hub cluster from a previous restore, it is not restored again. Even when the backup contains the expected updates, the policy contents are not updated by Velero on the new hub cluster. 
 
 To address the previously mentioned limitations, when a `restore.cluster.open-cluster-management.io` resource is created, the cluster backup and restore operator runs a set of steps to prepare for restore by cleaning the hub cluster before Velero restore begins. For more information, see link:../clusters/backup_and_restore.adoc#clean-hub-restore[Clean the hub cluster before restore].
+
+[#submariner-known-issues]
+== Submariner known issues
+
+[#submariner-no-rhel]
+=== Submariner network loses conenection with clusters that ues Red Hat Enterprise Linux nodes
```
Comment (dfarrell07) [inline release_notes/known_issues.adoc]: I think this only applies to worker nodes, not gateway nodes. Right @sridhargaddam?

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -930,3 +930,11 @@ View the following restore limitations:
 * The new hub cluster has a different configuration from the active hub cluster when a user reapplies the backup on the new hub cluster. Since there is an existing policy on the hub cluster from a previous restore, it is not restored again. Even when the backup contains the expected updates, the policy contents are not updated by Velero on the new hub cluster. 
 
 To address the previously mentioned limitations, when a `restore.cluster.open-cluster-management.io` resource is created, the cluster backup and restore operator runs a set of steps to prepare for restore by cleaning the hub cluster before Velero restore begins. For more information, see link:../clusters/backup_and_restore.adoc#clean-hub-restore[Clean the hub cluster before restore].
+
+[#submariner-known-issues]
+== Submariner known issues
+
+[#submariner-no-rhel]
+=== Submariner network loses conenection with clusters that use Red Hat Enterprise Linux nodes
```
Comment (sridhargaddam) [inline release_notes/known_issues.adoc]:
```suggestion
=== Submariner currently does not support Red Hat Enterprise Linux nodes as worker nodes.
```

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -930,3 +930,11 @@ View the following restore limitations:
...
+
+When you connect a Submariner network with at least one {ocp-short} cluster that uses Red Hat Enterprise Linux nodes, the connection fails when the cluster communicates with another cluster. Submariner only supports {ocp-short} clusters that use Red Hat Enterprise Linux CoreOS nodes.
```
Comment (sridhargaddam) [inline release_notes/known_issues.adoc]:
```suggestion
When deploying Submariner on a cluster that includes Red Hat Enterprise Linux worker nodes, application work-loads fail to communicate with remote clusters. Submariner only supports {ocp-short} clusters that use Red Hat Enterprise Linux CoreOS nodes.
```

File: release_notes/known_issues.adoc (hunk 1 above)
Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: Correct. I suggested a slightly different sentence.

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (am-agrawa) [reply release_notes/known_issues.adoc]: +1

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (am-agrawa) [reply release_notes/known_issues.adoc]: +1

**Comment by cadawson** [comment]
Update made @sridhargaddam . Thanks all!

File: release_notes/known_issues.adoc (hunk 3 above)
Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: @cadawson looks like this is missed.

**Comment by cadawson** [comment]
That should cover it, @sridhargaddam . Let me know if there are other updates. Thanks!

**Comment by sridhargaddam** [comment]
/lgtm

Thanks @cadawson

**Pull request merged**